Package.swift

```diff
@@ -42,14 +42,18 @@ let package = Package(
             name: "_StringProcessing",
             dependencies: ["_MatchingEngine"],
             swiftSettings: [
-                .unsafeFlags(["-enable-library-evolution"])
+                .unsafeFlags(["-enable-library-evolution"]),
+                .unsafeFlags(["-Xfrontend", "-enable-experimental-pairwise-build-block"])
```

Comment:
I'm not sure why Github shows these diffs given that they're merged. Do you think you can rebase?

Reply:
Yeah I'm in the process of rebasing. There's a lot of conflicts here.

Reply:
Hmm, would it be a rebase or just a cherry-pick of 41ffdc7af3?

Reply:
Phew... done.